### PR TITLE
Captain and Security Hardsuits can jetpack from suit storage

### PR DESCRIPTION
**Pull request by Robustin**
It's really common sense that what should be the most robust suits on station have a feature that at least lets them compete with Syndicate or Engineering hardsuits.

Why this PR is good:

1) Doesn't extend the internal jetpack thing, this change still requires a JETPACK to function, not just an internals tank. So no "free jetpacks" for anyone else. 

2) Lets the "combat" hardsuits actually be competitive in combat

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -71,7 +71,7 @@
 	var/safe_co2_min = 0
 	var/safe_co2_max = 10 // Yes it's an arbitrary value who cares?
 	var/safe_toxins_min = 0
-	var/safe_toxins_max = 0.05
+	var/safe_toxins_max = 0.005
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species.dm]: wee woo wee woo random thing reversion update your branch

Comment (Robustin) [inline code/modules/mob/living/carbon/human/species.dm]: what the actual fuck

**Comment by Zilenan91** [comment]
So what does this do, you're saying it doesn't expand internal jetpacks to more suits but it requires a jetpack to function so what

**Comment by Robustin** [comment]
It means you don't have to hold your bag in your hands like some kind of pleb when you jetpack, freeing it up for riot shields, off-hands, easier inventory management, etc.

**Comment by PKPenguin321** [comment]
:-1: these hardsuits are for enabling space exploration, jetpacks are for space combat. if you want the high armor HoS/captain suit for space combat you should have to use a jetpack as the drawback

**Comment by Robustin** [comment]
>Sec hardsuits are for space exploration
>Captain's hardsuit should be worse in combat than what an ordinary engineer gets

PKP you've got some top-notch memes fam.

**Comment by PKPenguin321** [comment]
if you just use a back mounted jetpack you're just as good to go if not better than an engi hardsuit you dingus

yes i do think these hardsuits are mostly for exploration (BY THEMSELVES, they are perfectly viable for combat with a back mounted jetpack), although that's less relevant now that away missions are gone. haven't you ever seen the space expedition things where they'll get each department in their own hardsuit and head out to explore?

**Comment by Robustin** [comment]
>PKP thinks the guy who has to hold most of his gear in a bag in his hand has an advantage in combat over the dudes who can swap inventory freely and doesn't lose most of his inventory if he gets stunned

If you don't think it's an advantage then why argue with me, it's just a silly meme change to you, no reason to oppose it. You said yourself in the first post that being forced to use the jetpack was a "disadvantage" relative to what engineers get.

**Comment by Robustin** [comment]
Conflicts? What? I'm not even getting conflicts when I pull upstream-master to this branch.

**Comment by Dorsisdwarf** [comment]
Remove the captains personal jetpack if you're giving him a built-in.

If this already does that, sorry, the code is completely unreadable due to dismemes

**Comment by Steelpoint** [comment]
From what I'm reading this PR does NOT give the Security or Captain's Hardsuits a inbuilt jetpack, but it simply allows them to place a Jetpack into the Suit Storage Slot and it'll still work, meaning you don't need to wear a jetpack on your Back slot.

So Security still needs to get a Jetpack from EVA.

**Comment by Robustin** [comment]
Nobody is getting a built-in, thats point #1 of this very short PR summary at the top. 

I considered giving the HOS one for his suit but:

1) Giving that its 1 regular wall away from space it would be more likely to get used against security

2) Kor and a few others hate jetpack power creep. So HoS will still have to schlep to EVA and beg the AI or laser his way in.

**Comment by kevinz000** [comment]
implantable thruster set

**Comment by KorPhaeron** [comment]
You broke everything and also I already said why I dislike this.

I'm gonna go nerf inbuilt jetpacks wish me luck!

**Comment by KorPhaeron** [comment]
I guess I'm actually in the mood to code something I'll only get yelled at for anyway.

Internal hardsuit jetpacks should work like helmets though, and deploy onto the back. They should be pulse pressurized like syndiborg jetpacks are.

This PR is just more inventory/power creep "quality of life"

**Comment by PKPenguin321** [comment]
^

**Comment by KorPhaeron** [comment]
Doesn't compile, merge conflicts, don't like the inventory creep.

Inbuilt jetpacks were a mistake, they shouldn't be used to justify further mistakes

**Pull request closed without merging**